tests/Storage/MongoDBStorageAdapterTest.php

```diff
@@ -66,7 +66,15 @@ public function testGetCollectionName()
 
     public function mockCollection()
     {
-        return $this->createMock('MongoCollection', ['find', 'findOne', 'update', 'remove'], [], '', false);
```

Comment:
Note that `createMock()` only accepts a single argument, the name of the class we wish to create a test double for.